src/esn.py

```diff
@@ -178,9 +394,9 @@ def _ridge_regression(self, xt: Array, ut: Array, yt_hat: Array,
             w_out = np.dot(np.linalg.pinv(S), yt_hat).T
         else:
             # ridge regression
-            R = np.dot(S.T, S) / xt.shape[0]
+            R = np.dot(S.T, S) #/ xt.shape[0]
```

Comment:
remove comments here

Reply:
Why do you normalize for ridge regression? It is not a correlation matrix